### PR TITLE
Adding missing DMFG quincy suites

### PR DESCRIPTION
**Pull request by tintumathew10**
# Description

Adding missing DMFG quincy suites.

### CODE REVIEW

File: suites/quincy/cephadm/test-haproxy-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,266 @@
+#===============================================================================================
+# Tier-level: 1
+# Test-Suite: tier-1_ssl_dashboard_port.yaml
+# Test-Case:
+#      Bootstrap cluster with multiple admin nodes and apply-spec options.
+#      Deploy HA proxy service and verify service disable, enable, restart options.
+#
+# Cluster Configuration:
+#   cephci/conf/pacific/cephadm/tier-1_5node_cephadm_bootstrap.yaml
```
Comment (vamahaja) [inline suites/quincy/cephadm/test-haproxy-deployment.yaml:9]: `pacific` ???

File: suites/quincy/cephadm/tier-0_5-1_cephadm.yaml [hunk 2]
```diff
@@ -0,0 +1,819 @@
...
+# Tier-0 test cases for
+#    - Bootstrap
+#    - Host management
+#      - Add/Remove nodes
+#      - SetAddress
+#      - Attach/remove node labels
+#    - Ceph role Service deployment with multiple options,
+#    - Removal of services
+#
+# Conf file - conf/pacific/cephadm/sanity-cephadm.yaml
```
Comment (vamahaja) [inline suites/quincy/cephadm/tier-0_5-1_cephadm.yaml:18]: `pacific` ?

File: suites/quincy/cephadm/tier-0_cephadm.yaml [hunk 3]
```diff
@@ -0,0 +1,802 @@
+#===============================================================================================
+#-------------------------------------
+#----- Ceph code changes in 5.0 ------
+#--------------------------------------
+#--------------------------------------
+# Conf: conf/pacific/cephadm/sanity-cephadm.yaml
```
Comment (vamahaja) [inline suites/quincy/cephadm/tier-0_cephadm.yaml:6]: `pacific` ?

File: suites/quincy/upgrades/tier-1_upgrade_cephadm.yaml [hunk 4]
```diff
@@ -0,0 +1,245 @@
+#===============================================================================================
+# Tier-level: 1
+# Test-Suite: tier-1_cephadm_upgrade.yaml
+# Test-Case: Upgrade ceph cluster from Build-to-Build (B2B)
+#
+# Cluster Configuration:
+#    conf/pacific/upgrades/tier-1_upgrade_cephadm.yaml
```
Comment (vamahaja) [inline suites/quincy/upgrades/tier-1_upgrade_cephadm.yaml:7]: `pacific` ?

File: suites/quincy/upgrades/tier-1_upgrade_cephadm_without_rgw.yaml [hunk 5]
```diff
@@ -0,0 +1,213 @@
+#===============================================================================================
+# Tier-level: 1
+# Test-Suite: tier-1_cephadm_upgrade.yaml
+# Test-Case: Upgrade ceph cluster from Build-to-Build (B2B)
+#
+# Cluster Configuration:
+#    conf/pacific/upgrades/tier-1_upgrade_cephadm_without_rgw.yaml
```
Comment (vamahaja) [inline suites/quincy/upgrades/tier-1_upgrade_cephadm_without_rgw.yaml:7]: `pacific` ?

File: suites/quincy/cephadm/test-haproxy-deployment.yaml [hunk 6]
```diff
@@ -0,0 +1,266 @@
+#===============================================================================================
+# Tier-level: 1
+# Test-Suite: tier-1_ssl_dashboard_port.yaml
```
Comment (vamahaja) [inline suites/quincy/cephadm/test-haproxy-deployment.yaml:3]: incorrect testsuite name

File: suites/quincy/cephadm/tier-0_5-1_cephadm.yaml [hunk 7]
```diff
@@ -0,0 +1,819 @@
+#===============================================================================================
+#-------------------------------------
+#----- Ceph code changes in 5.1 ------
```
Comment (vamahaja) [inline suites/quincy/cephadm/tier-0_5-1_cephadm.yaml:3]: `5.1` ?

File: suites/quincy/cephadm/tier-0_cephadm.yaml [hunk 8]
```diff
@@ -0,0 +1,802 @@
+#===============================================================================================
+#-------------------------------------
+#----- Ceph code changes in 5.0 ------
```
Comment (vamahaja) [inline suites/quincy/cephadm/tier-0_cephadm.yaml:3]: `5.0` ?

File: suites/quincy/upgrades/tier-1_upgrade_cephadm.yaml [hunk 9]
```diff
@@ -0,0 +1,245 @@
+#===============================================================================================
```
Comment (vamahaja) [inline suites/quincy/upgrades/tier-1_upgrade_cephadm.yaml:1]: update test suite name to `tier1-upgrade-cephadm.yaml`

File: suites/quincy/upgrades/tier-1_upgrade_cephadm_without_rgw.yaml [hunk 10]
```diff
@@ -0,0 +1,213 @@
+#===============================================================================================
```
Comment (vamahaja) [inline suites/quincy/upgrades/tier-1_upgrade_cephadm_without_rgw.yaml:1]: Update testsuite name to `tier1-upgrade-cephadm-without-rgw.yaml`

File: suites/quincy/cephadm/tier-0_cephadm.yaml [hunk 11]
```diff
@@ -0,0 +1,802 @@
+#===============================================================================================
```
Comment (vamahaja) [inline suites/quincy/cephadm/tier-0_cephadm.yaml:1]: Update testsuite name to `tier0-cephadm.yaml`

File: suites/quincy/cephadm/tier-0_5-1_cephadm.yaml [hunk 12]
```diff
@@ -0,0 +1,819 @@
+#===============================================================================================
```
Comment (vamahaja) [inline suites/quincy/cephadm/tier-0_5-1_cephadm.yaml:1]: Is this same testsuite as `cephadm/tier-0_cephadm.yaml` ?

File: suites/quincy/cephadm/test-haproxy-deployment.yaml [hunk 13]
```diff
@@ -0,0 +1,266 @@
+#===============================================================================================
```
Comment (vamahaja) [inline suites/quincy/cephadm/test-haproxy-deployment.yaml:1]: Update name to `tier1-haproxy-deployment.yaml`

File: conf/quincy/cephadm/sanity-cephadm.yaml [hunk 14]
```diff
@@ -0,0 +1,68 @@
+globals:
```
Comment (vamahaja) [inline conf/quincy/cephadm/sanity-cephadm.yaml:1]: Update config name to `1admin-8node-1client-cluster.yaml`

File: conf/quincy/upgrades/tier-1_upgrade_cephadm.yaml [hunk 15]
```diff
@@ -0,0 +1,42 @@
+globals:
```
Comment (vamahaja) [inline conf/quincy/upgrades/tier-1_upgrade_cephadm.yaml:1]: Update name to `1admin-3node-1client-cluster.yaml`

**Review by vamahaja — CHANGES_REQUESTED** [review]

File: suites/quincy/cephadm/test-haproxy-deployment.yaml (hunk 13 above)
Comment (tintumathew10) [reply suites/quincy/cephadm/test-haproxy-deployment.yaml:1]: done

File: conf/quincy/cephadm/sanity-cephadm.yaml (hunk 14 above)
Comment (tintumathew10) [reply conf/quincy/cephadm/sanity-cephadm.yaml:1]: done

File: suites/quincy/cephadm/tier-0_5-1_cephadm.yaml (hunk 12 above)
Comment (tintumathew10) [reply suites/quincy/cephadm/tier-0_5-1_cephadm.yaml:1]: cephadm/tier-0_cephadm.yaml - this suite has most of the test cases from tier-0_5-1_cephadm.yaml. Added the missing tests from tier-0_5-1_cephadm.yaml to tier-0_cephadm.yaml suite and remove the duplicated suite.

File: suites/quincy/cephadm/tier-0_cephadm.yaml (hunk 11 above)
Comment (tintumathew10) [reply suites/quincy/cephadm/tier-0_cephadm.yaml:1]: done

File: suites/quincy/upgrades/tier-1_upgrade_cephadm_without_rgw.yaml (hunk 10 above)
Comment (tintumathew10) [reply suites/quincy/upgrades/tier-1_upgrade_cephadm_without_rgw.yaml:1]: done

File: suites/quincy/cephadm/tier-0_cephadm.yaml (hunk 8 above)
Comment (tintumathew10) [reply suites/quincy/cephadm/tier-0_cephadm.yaml:3]: removed

File: suites/quincy/cephadm/test-haproxy-deployment.yaml (hunk 6 above)
Comment (tintumathew10) [reply suites/quincy/cephadm/test-haproxy-deployment.yaml:3]: removed

File: suites/quincy/upgrades/tier-1_upgrade_cephadm_without_rgw.yaml (hunk 5 above)
Comment (tintumathew10) [reply suites/quincy/upgrades/tier-1_upgrade_cephadm_without_rgw.yaml:7]: updated

File: suites/quincy/upgrades/tier-1_upgrade_cephadm.yaml (hunk 4 above)
Comment (tintumathew10) [reply suites/quincy/upgrades/tier-1_upgrade_cephadm.yaml:7]: changed

File: suites/quincy/cephadm/tier-0_5-1_cephadm.yaml (hunk 2 above)
Comment (tintumathew10) [reply suites/quincy/cephadm/tier-0_5-1_cephadm.yaml:18]: changed

File: suites/quincy/cephadm/test-haproxy-deployment.yaml (hunk 1 above)
Comment (tintumathew10) [reply suites/quincy/cephadm/test-haproxy-deployment.yaml:9]: changed

**Review by vamahaja — APPROVED** [review]

**Pull request merged**